src/AddTodoForm.jsx

```diff
@@ -1,32 +1,26 @@
-
-
 import React, { useState } from 'react';
 
-const AddTodoForm = ({ onAddTodo }) => {
-
-  const [todoTitle, setTodoTitle] = React.useState('');
-
-  // new Function
-  const handleAddTodo = (event) => {
-    // hint
-    event.preventDefault();
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState('');
 
-    const todoTitle = event.target.title.value;
-
-    console.log(todoTitle);
-
-    event.target.reset();
-
-    // last sentence of the part that says app,invoke the onAddTodo
-    onAddTodo(todoTitle);
+  const handleTitleChange = (event) => {
+    setTodoTitle(event.target.value);
+  };
 
+  const handleAddTodo = () => {
+    const newTodo = {
+      id: Date.now(), //identificador único temporal
+      title: todoTitle
+    };
+    onAddTodo(newTodo);
+    setTodoTitle('');
   };
 
   return (
-    <form onSubmit={handleAddTodo}>
-      <input type="text" name="title" />
-      <button type="submit">Add</button>
-    </form>
+    <div>
+      <input type="text" value={todoTitle} onChange={handleTitleChange} />
```

Comment:
There is a missing element that gives meaning to your `input`.

Reply:
`      <form onSubmit={handleAddTodo}> 
        <label htmlFor="todoTitle">Fill: </label>
        <input type="text" id="todoTitle" name="todoTitle" value={todoTitle} onChange={handleTitleChange} required/>
        <button type="submit">Add Todo</button>
      </form> `